app/models/product_drive_participant.rb

```diff
@@ -31,6 +31,10 @@ def volume
     donations.map { |d| d.line_items.total }.reduce(:+)
   end
 
+  def volume_by_product_drive(product_drive_id)
+    donations.by_product_drive(product_drive_id).map { |d| d.line_items.total }.sum
```

Comment:
There is almost definitely a more performant way to do this via SQL. Not sure if there tend to be more than a handful of product drive participants to be shown. @cielf ?

Reply:
Let's see how this goes, but I think this should be fine at the current scale. We would see slow load times on the page otherwise and can address them then.

Reply:
We have a total of ~17000 donations for all time

Reply:
And about 1/10 the number of product drive participants.